packages/react-dom/src/client/ReactDOMFiberComponent.js

```diff
@@ -465,42 +496,40 @@ export function setInitialProperties(
   switch (tag) {
     case 'iframe':
     case 'object':
-      trapBubbledEvent('topLoad', 'load', domElement);
+      trapBubbledEvent(TOP_LOAD, 'load', domElement);
```

Comment:
We can remove all of these event name strings like `'load'` if we have `trapBubbledEvent`/`trapCaptureEvent` call `getRawEventName` directly. Then you only need to call it with the